book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+  end
+end
```

Comment:
- [ ] you have to create a new instance of rental class since you have already handled the push in rental initialize
 ```
def add_rental(person, date)
    Rental.new(date, person, self)
  end
```